src/pages/RegisterReferralSendDate.tsx

```diff
@@ -0,0 +1,50 @@
+import {useTranslation} from "react-i18next";
+import {useState} from "react";
+import {namespaces} from "../i18n/i18n.constants";
+import AppReferralSendDateForm from "../components/molecules/AppReferralSendDateForm";
+import AppPageHeader from "../components/molecules/AppPageHeader";
+import AppWrapper from "../components/molecules/AppWrapper";
+import {AppRoute} from "../enums/app-route";
+import AppMessageDialog from "../components/molecules/AppMessageDialog";
+import AlertIcon from "../components/icons/AlertIcon";
+
+const RegisterReferralSendDate = () => {
+    const {t: translate} = useTranslation(namespaces.pages.registerReferralSendDate);
+    const [showConfirmDialog, setshowConfirmDialog] = useState(false);
```

Comment:
```suggestion
    const [showConfirmDialog, setShowConfirmDialog] = useState(false);
```

Reply:
Nice catch again! I definitely didn't see that